src/traffic_server/HostStatus.cc

```diff
@@ -113,6 +137,14 @@ HostStatus::~HostStatus()
   ink_rwlock_destroy(&host_statids_rwlock);
 }
 
+void
+HostStatus::loadStats()
+{
+  if (RecLookupMatchingRecords(RECT_ALL, stat_prefix.c_str(), handle_record_read, nullptr) != REC_ERR_OKAY) {
+    Error("[HostStatus] - Error reading stats.");
```

Comment:
Maybe the error should say that error while readings for updating the host status?